src/game/game.cpp

```diff
@@ -2,5 +2,13 @@
 
 namespace game
 {
+	void SV_GameDropClient(int clientNum, const char* reason)
+	{
+		assert(sv_maxclients->current.integer >= 1 && sv_maxclients->current.integer <= 18);
```

Comment:
Why this?

Reply:
It's a check to make sure that sv_maxclients is not out of bounds. I just replicated the game's original function.
If you don't want it I can remove it

Reply:
Don't really see the point of it since the dvar already has limits so i guess remove it

Reply:
Removed